stdlib/enum.pyi

```diff
@@ -181,9 +181,11 @@ class Enum(metaclass=EnumMeta):
 
 if sys.version_info >= (3, 11):
     class ReprEnum(Enum): ...
-    _IntEnumBase = ReprEnum
+
+if sys.version_info >= (3, 11):
+    _IntEnumBase: TypeAlias = ReprEnum
 else:
-    _IntEnumBase = Enum
+    _IntEnumBase: TypeAlias = Enum
```

Comment:
I'm not sure about using `TypeAlias` here. `_IntEnumBase` is not used as a type here, but as a regular alias.

Reply:
Reverted :)